templates/behavior.yml

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
can you put this file in demo_templates? it may be confusing to have two template folders. If they are only used in test, you can put it under `testdata`, that special folder name is ignored in the compilation step in build and test.

Reply:
actually wasn't meant to be committed, oops. will remove

Reply:
i think it should be part of the demo_templates, moved there, to reflect that thing.